src/tabs/tab.jsx

```diff
@@ -125,6 +148,7 @@ const Tab = React.createClass({
         onTouchTap={this._handleTouchTap}
       >
         {label}
+        {iconElement}
```

Comment:
Shouldn't we put this above the `{label}`?
![capture d ecran 2016-01-25 a 13 39 37](https://cloud.githubusercontent.com/assets/3165635/12551223/375e2b6c-c369-11e5-8132-b1d05292b794.png)